### ISSUE TITLE
Rule Proposal: recommended attributes in html tag

### ISSUE THREAD

**Issue by Cellule** [yannickcr/eslint-plugin-react#103]
I am not sure if this falls in the scope of this plugin, but I while I was working today I saw react-a11y warn me about missing `alt` in an `<img />`.
Now I wanted to fix these problems and I thought it would be nice if my linter could point them to me directly in my editor.

**Comment by mathieumg** [comment]
Somewhat related: https://github.com/rackt/react-a11y/issues/25

**Comment by lencioni** [comment]
@Cellule you should check out [eslint-plugin-jsx-a11y](https://github.com/evcohen/eslint-plugin-jsx-a11y)! 

I think that a specialized accessibility eslint plugin is best for this, so I think this issue should probably be closed out.

**Comment by ljharb** [comment]
Agreed.

**Comment by Cellule** [comment]
Thanks for sharing this, this is exactly what I was looking for.
